spec/index.html

```diff
@@ -1053,17 +1052,11 @@ <h3>RDF Dataset Comparison</h3>
       graph <var>NG1</var> and the RDF dataset <var>D2</var> with default graph
       <var>DG2</var> and any named graph <var>NG2</var>)
       are <dfn data-lt="dataset isomorphism" class="export">dataset-isomorphic</dfn> if and only if
-      there is a bijection <var>M</var> between the nodes, triples and graphs in
-      <var>D1</var> and those in <var>D2</var> such that all of the following properties hold:</p>
+      there is a bijection <var>M</var> from [=blank nodes=] to [=blank nodes=],
```

Comment:
NG1 can be a blank node.

Reply:
@gkellogg I don't understand your comment.

That being said, I reused the language about NG1 and NG2 as is, but I find it quite confusing. I'll change it into something that I find more understandable. Hopefully it will solve your comment above as well :)

Reply:
My reading of this is that it covers isomorphism of terms within the default and named graphs, but not on the graph names, themselves. The graph names also need to be part of _M_.

Reply:
The previous wording was indeed ambiguous, as it said "bijection M between nodes, triples and graphs", sounding like graph names were excluded. Actually, there were *not* excluded, as later it said:

> <n, G> is in NG1 if and only if <M(n), M(G)> is in NG2. 

so graph named are also mapped via M.

And the proposed wording in this PR (as well as Olaf's counterproposal in #160) is described as mapping from *terms* to *terms*, regardless on where those terms are used. So hopefully, this should be even clearer that graph names are also mapped.